src/cocotb_tools/runner.py

```diff
@@ -80,16 +92,14 @@ def __init__(self) -> None:
         self.build_dir: Path = get_abs_path("sim_build")
         self.parameters: Mapping[str, object] = {}
 
-    @abc.abstractmethod
+    @abstractmethod
     def _simulator_in_path(self) -> None:
         """Raise exception if the simulator executable does not exist in :envvar:`PATH`.
 
         Raises:
             SystemExit: Simulator executable does not exist in :envvar:`PATH`.
         """
 
```

Comment:
Is there a reason `raise NotImplementedError()` was removed from some but not the other abstract methods?